RELEASES.md

```diff
@@ -1,3 +1,202 @@
+Version 1.26.0 (2018-05-10)
+==========================
+
+Language
+--------
+- [Closures now implement `Copy + Clone` if all captured variables do.][49299]
```

Comment:
I read this as meaning that both `Copy` and `Clone` have to be implemented, thus `Copy + Clone`, but I think this change is also effective if the captured variables are only `Clone`.